src/main/java/uk/gov/ons/ctp/integration/mockcaseapiservice/CasesConfig.java

```diff
@@ -24,8 +22,6 @@
 @ConfigurationProperties("casedata")
 public class CasesConfig {
 
-  private LuhnCheckDigit luhnChecker = new LuhnCheckDigit();
```

Comment:
I don't understand why you have removed the LuhnChecker. I put this in because it is an easy up-front way to prevent us putting in bad data, which for the RM simulator will never happen. I believe this will save time for us.

Reply:
Yes that was a little rash of me sorry. I have added it back in now.